ClientApp/src/NavBar/NavBar.scss

```diff
@@ -0,0 +1,100 @@
+@import '../Styles/constants';
+
+ul {
```

Comment:
These are going to target ul and li elements in the whole codebase (contrary to what it might seem that we're only in the NavBar.scss file) so it might be worth giving these classnames and targetting specific classnames here instead

Reply:
We've added some classnames to our scss file :)